.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 ruby:
```

Comment:
instead of `ruby`

try `rvm`

https://docs.travis-ci.com/user/languages/ruby